Dockerfile.moc-xdmod

```diff
@@ -62,7 +62,7 @@ WORKDIR /root
 RUN mkdir -p /root/rpmbuild/RPMS/noarch \
     && mkdir /tmp/supremm \
     && pip3 install --upgrade pip \
-    && pip3 install keystoneauth python-keystoneclient python-novaclient python-cinderclient cryptography python-openstacksdk mysql-connector-python pexpect
+    && pip3 install keystoneauth python-keystoneclient python-novaclient python-cinderclient cryptography python-openstacksdk mysql-connector-python==8.0.29 pexpect
```

Comment:
With all these requirements, it would be better to move them into a `requirements.txt` file in the repository.